configuration/logging.md

```diff
@@ -28,13 +28,58 @@ log.console.level = debug | info | warning | error
 
 ### Error Logging
 
+VerneMQ log error messages by default. One can change the default behaviour by setting:
+
+```text
+log.error = on | off
+```
+
 VerneMQ defaults to log the error messages to a file, which can specified by:
 
 ```text
 log.error.file = /path/to/log/file
 ```
 
-This option defaults to the filename `error.log`, whereas the path differs on the way VerneMQ is installed.
+This option defaults to `/var/log/vernemq/error.log` for Ubuntu, Debian, RHEL and Docker installs.
+
+### Crash Logging
+
+VerneMQ log crash messages by default. One can change the default behaviour by setting:
+
+```text
+log.crash = on | off
+```
+
+VerneMQ defaults to log the crash messages to a file, which can specified by:
+
+```text
+log.crash.file = /path/to/log/file
+```
+
+This option defaults to `/var/log/vernemq/crash.log` for Ubuntu, Debian, RHEL and Docker installs.
+
+The maximum sizes in bytes of inidividual messages in the crash log defaults to `64KB` but can be specified by:
+
+```text
+log.crash.maximum_message_size = 64KB
+```
+
+VerneMQ rotate crash logs. By default, the crash log file is rotated at midnight or when the size exceeds `10MGB`. This behaviour can be changed by setting:
+
+```text
+## Acceptable values:
+##   - a byte size with units, e.g. 10GB
+log.crash.size = 10MB
+
+## For acceptable values see https://github.com/basho/lager/blob/master/README.md#internal-log-rotation
```

Comment:
This link will be probably `https://github.com/erlang-lager/lager/blob/master/README.md#internal-log-rotation`